client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -128,11 +128,17 @@
                   icon="ai"
                   size="large"
                   :text="Translator.trans('segment.oracle.tooltip')" />
-                <dp-badge
-                  v-if="activeId === 'oracleRec'"
-                  class="absolute u-right-0 u-mr"
-                  size="smaller"
-                  :text="Translator.trans('segment.oracle.beta')" />
+                <span
```

Comment:
Out of curiosity: why didn't it work to put the tooltip directly onto the DpBadge component?

Reply:
I was not sure whether the badge element should include a tooltip inside. I initially considered it as more of a specific case.  If you think that it would be better to put the tooltip inside the badge, I can do it. Now, I believe it's a better idea...

The implementation could be similar to [this one](https://github.com/demos-europe/demosplan-core/blob/22e6a44496bd491cab280d8578bc83b3f33a164d/client/js/components/map/map/DpOlMapEditFeature.vue#L210-L217)